transactions/hybrid-custody/onboarding/blockchain_native.cdc

```diff
@@ -64,22 +64,22 @@ transaction(
         var acctCap = newAccount.linkAccount(HybridCustody.LinkedAccountPrivatePath)
             ?? panic("problem linking account Capability for new account")
 
-        // Create a ChildAccount & link Capabilities
-        let ChildAccount <- HybridCustody.createChildAccount(acct: acctCap)
-        newAccount.save(<-ChildAccount, to: HybridCustody.ChildStoragePath)
+        // Create a OwnedAccount & link Capabilities
+        let OwnedAccount <- HybridCustody.createChildAccount(acct: acctCap)
+        newAccount.save(<-OwnedAccount, to: HybridCustody.ChildStoragePath)
         newAccount
-            .link<&HybridCustody.ChildAccount{HybridCustody.BorrowableAccount, HybridCustody.ChildAccountPublic, HybridCustody.ChildAccountPrivate}>(
+            .link<&HybridCustody.OwnedAccount{HybridCustody.BorrowableAccount, HybridCustody.OwnedAccountPublic}>(
                 HybridCustody.ChildPrivatePath,
                 target: HybridCustody.ChildStoragePath
             )
         newAccount
-            .link<&HybridCustody.ChildAccount{HybridCustody.ChildAccountPublic}>(
+            .link<&HybridCustody.OwnedAccount{HybridCustody.OwnedAccountPublic}>(
```

Comment:
Also wondering if we should link `MetadataViews.Resolver` here, but also out of scope for this PR. Just a note if we think that should change.

Reply:
It's a good question. I think, in general, metadata is considered to be public which is in-line with how NFTs are organized